etools-searchable-multiselection-menu.html

```diff
@@ -659,7 +691,8 @@
         },
 
         _singleValueSelected: function() {
-          this.options.forEach(function(item) {
+          var options = this.twoLines ? this.twoLinesOptions: this.options;
```

Comment:
twoLineOptions is necessary because there is an observer on options, if I set these values to options, you get a max stack size exceeded exception.